src/azure/Model/MethodPya.cs

```diff
@@ -55,25 +55,24 @@ public override string ExceptionDocumentation
         {
             get
             {
-                if (DefaultResponse.Body == null || DefaultResponse.Body.Name == "CloudError")
+                if (DefaultResponse.Body == null || DefaultResponse.Body.Name == "HttpResponseError")
                 {
-                    return ":class:`CloudError<msrestazure.azure_exceptions.CloudError>`";
+                    return ":class:`HttpResponseError<azure.core.HttpResponseError>`";
```

Comment:
This needs discussion here: CloudError is more than a simple error, it's an ARM oDataV4 parse integrated. Shifting that to a simple exception is a huge loss.
@johanste